BootloaderCommonPkg/Include/Guid/LoaderPlatformInfoGuid.h

```diff
@@ -46,6 +46,10 @@ extern EFI_GUID gLoaderPlatformInfoGuid;
 //Secure debug (0 - disabled, 1 - enabled)
 #define        HWSTATE_SECURE_DEBUG              BIT3
 
+//Hw Platform debug (0 - disabled, 1 - enabled)

```

Comment:
Can we add more description around this i.e. under what circumstances it's set vs un-set?